requirements/time_tracker_api/dev.txt

```diff
@@ -15,4 +15,7 @@ pytest-mock==2.0.0
 coverage==4.5.1
 
 # Git hooks
-pre-commit==2.2.0
\ No newline at end of file
+pre-commit==2.2.0
+
+#Azure functions
+azure-functions==1.6.0
```

Comment:
This library is included when you run `pip install -r requirements/time-tracker-events/dev.txt`. No need to put it here. Maybe we should update the Readme to mention this, but that would be another ticket.

Reply:
I didn't see what time tracker events is included in it, I'm going to remove it. But I think it is important to mention it in another ticket to update the README

Reply:
The call to `pip install -r requirements/time-tracker-events/dev.txt` makes a call to `requirements/time-tracker-events/prod.txt` which makes another call to `-r ../../time_tracker_events/requirements.txt` that is where this library is included.

Reply:
You are right, thanks for the observation